### ISSUE TITLE
The integration astroweather is using the deprecated forecast attribute.

### ISSUE THREAD

**Issue by heinemannj** [mawinkler/astroweather#32]
After upgrading of HA to

```
Home Assistant 2023.9.0
Supervisor 2023.08.3
Operating System 10.5
Frontend 20230906.1 - latest
```

the following waring pop's up:

`The integration astroweather is using the deprecated forecast attribute.`

Please check:

https://www.home-assistant.io/blog/2023/09/06/release-20239/#weather-forecast-service

**Comment by BaltasarParreira** [comment]
Hi, yes I started getting the same warning after install the lasted core (not a beta one).

**Comment by marwinsworld** [comment]
You're both correct, I missed that. Will take care of that.

**Comment by Snakebiteuk** [comment]
> You're both correct, I missed that. Will take care of that.

Perfect just updated to see the same.
Thank you.

**Comment by villaniclaudio8** [comment]
Hi, I wanted to know if the correction has been made for the message you receive after updating Home Assistant to 9.0.
I tried to uninstall in Integration of HACS Astroweather reboot and then reinstall it but the message astroweather uses deprecated forecast attribute still has the error. What can be done to remedy the problem? Thank you

**Comment by heinemannj** [comment]
> Hi, I wanted to know if the correction has been made for the message you receive after updating Home Assistant to 9.0. I tried to uninstall in Integration of HACS Astroweather reboot and then reinstall it but the message astroweather uses deprecated forecast attribute still has the error. What can be done to remedy the problem? Thank you

Maby you have to wait, until our famous developers take there private time/efforts to help the community with requirerd fixes???

**Comment by Davvy80** [comment]
Same notice

**Comment by fuzzysb** [comment]
So see that from [Here](https://www.home-assistant.io/blog/2023/09/06/release-20239/#weather-forecast-service) there is a new service to get the forecast, the forecast attribute below is no longer supported:

![image](https://github.com/mawinkler/astroweather/assets/16162689/6398bcb1-8db1-4982-acb8-8a1599e01aff)

I have been reviewing the code, if i can help any maybe share branch with WIP, i can see if i can also assist

**Comment by mawinkler** [comment]
I'll fix this as time permits. Currently, it's just a deprecation warning and will continue to work until the release 2024.3.

**Comment by mawinkler** [comment]
You might test it. Deploy from the main branch, since I haven't made a new release yet.

**Comment by ejpenney** [comment]
I just installed from main and everything looks great, all the data is still present and the repair message is gone.

**Comment by B4dC4rd** [comment]
Still have the warning on my side

**Comment by heinemannj** [comment]
> You might test it. Deploy from the main branch, since I haven't made a new release yet. It's just a quick fix to provide the service and still be compatible with the lovelace card.

The replacement of `weather.py` and a restart of HA has removed the warning.

Many thanks!

**Comment by fuzzysb** [comment]
no warning for me any more either

**Comment by flight82** [comment]
> You might test it. Deploy from the main branch, since I haven't made a new release yet. It's just a quick fix to provide the service and still be compatible with the lovelace card.

It worked for me.
Thanks!

**Comment by Davvy80** [comment]
i don't understand the solution which is, how i can make? somebody can splain

**Comment by ejpenney** [comment]
> i don't understand the solution which is, how i can make? somebody can splain

The dev was looking for folks to test a change, if you don't know how to do that it's probably just best to wait for the release which, based on the feedback, I would suspect is coming soon.

**Comment by Davvy80** [comment]
> > i don't understand the solution which is, how i can make? somebody can splain
> 
> The dev was looking for folks to test a change, if you don't know how to do that it's probably just best to wait for the release which, based on the feedback, I would suspect is coming soon.

Very gentleman i'll waiting next update, thank you

**Comment by mdisabato** [comment]
I'll add my confirmation to the fix working. No warning and all the data is present. 

Thank you so much!
:)

**Comment by mawinkler** [comment]
Thank you all for testing. Just released [v0.23.2](https://github.com/mawinkler/astroweather/compare/v0.23.1...v0.23.2).
